src/Controller/MapController.php

```diff
@@ -13,8 +13,19 @@ class MapController extends AbstractController
      */
     public function index(): Response
     {
+        $user = $this->getUser();
+        $eventsOrganized = $user->getEventOrganized();
+        $events = [];
+        foreach ($eventsOrganized as $eventOrganized) {
+            $events[] = $eventOrganized;
+        }
+        $eventsGoing = $user->getEventGoing();
+        foreach ($eventsGoing as $eventGoing) {
+            $events[] = $eventGoing;
+        }
```

Comment:
try this:

    $events = array_merge($user->getEventOrganized(), $user->getEventGoing());